packages/core-mobile/app/services/bridge/UnifiedBridgeService.ts

```diff
@@ -32,43 +40,82 @@ export class UnifiedBridgeService {
 
   // init and fetch configs
   async init({
-    environment,
-    disabledBridgeTypes
+    listenerApi
   }: {
-    environment: Environment
-    disabledBridgeTypes: BridgeType[]
-  }): Promise<void> {
+    listenerApi: AppListenerEffectAPI
+  }): Promise<boolean> {
+    const state = listenerApi.getState()
+
+    if (this.isInitialized()) {
+      const prevState = listenerApi.getOriginalState()
+
+      if (!this.shouldReinitializeBridge(prevState, state)) return false
+    }
+
+    const featureGates = this.getFeatureGateStates(state)
+    const isTest = featureGates.isDeveloperMode
+    const enabledBridgeTypes = this.getEnabledBridgeTypes(featureGates)
+
+    const environment = isTest ? Environment.TEST : Environment.PROD
+    const request = createInAppRequest(listenerApi.dispatch)
+
     Logger.info('initializing unified bridge service', {
-      environment,
-      disabledBridgeTypes
+      environment
     })
 
-    this.service = createUnifiedBridgeService({
-      environment,
-      disabledBridgeTypes
-    })
+    const bitcoinProvider = await getBitcoinProvider(isTest)
+
+    const evmSigner: EvmSigner = {
+      sign: async ({ data, from, to, chainId }) => {
```

Comment:
you can make evmSigner and btcSigner params of init

Reply:
fixed.